Configuration/StandardSequences/python/RawToDigi_cff.py

```diff
@@ -108,6 +108,7 @@ def _modifyRawToDigiForStage1Trigger( theProcess ) :
     """
     Modifies the RawToDigi sequence if using the Stage 1 L1 trigger
     """
+    theProcess.load("EventFilter.RctRawToDigi.l1RctHwDigis_cfi")
```

Comment:
@nsmith-  could you start a new cff that has all of the L1Trigger items in it? (to define L1RawToDigiSeq).